controllers/usersignup/usersignup_controller.go

```diff
@@ -224,6 +224,11 @@ func (r *Reconciler) handleDeactivatedUserSignup(logger logr.Logger, config tool
 		return reconcile.Result{}, err
 	}
 
+	err = r.updateStatus(logger, userSignup, r.setStatusDeactivatedApproval)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
```

Comment:
I would rather do the change of both conditions at the same time - doing two updates of the Status immediately after each other is not needed.
Could you please update the `setStatusDeactivated` so it sets both conditions?

Reply:
+1

Reply:
Done